project-extender/src/providers/jira-cloud-provider/JiraCloudProvider.ts

```diff
@@ -1,11 +1,23 @@
 /* eslint-disable max-classes-per-file */
 /* eslint-disable class-methods-use-this */
+import fetch from "cross-fetch"
 import { ProviderApi, ProviderCreator } from "../base-provider"
+import { Project } from "../base-provider/schema"
 import { getAccessToken } from "./getAccessToken"
 
 class JiraCloudProvider implements ProviderApi {
+  logout(): Promise<void> {
+    return new Promise((resolve, reject) => {
+      resolve()
+
+      reject()
+    })
+  }
```

Comment:
This promise will always resolve. That means the reject has no effect here. Logout doesn't only refer to logging out from the jira instance. We also need to logout from our provider, in this case it means setting the access token to undefined. So that the method `isLoggedIn()` returns `false`. And then simply resolving. We don't need to reject, as no error can happen here.

Reply:
And please move this under the class properties. For readability, it is best to have the class properties on top and then the methods.